crates/chisel/src/dispatcher.rs

```diff
@@ -528,7 +528,22 @@ impl ChiselDispatcher {
                                         err.name,
```

Comment:
omg why does this exist. We have ABI to solidity in cast and forge already, and a whole crate to interface with etherscan (foundry-block-explorers), please just use those to fix a trillion other bugs

Reply:
this definitely needs a cleanup pass but should happen in a followup